### PR TITLE
Remove final access modifier from public classes

### PR DESCRIPTION
**Pull request by jasonm23**
To facilitate tests/mocking, magnet should not use the `final` modifier.

This PR only removes the `final` keyword, and nothing else.

(Incoming PR for Sauce to do the same.)

### CODE REVIEW

File: Lib/Magnet/HotKey.swift [hunk 1]
```diff
@@ -11,7 +11,7 @@
 import Cocoa
 import Carbon
 
-public final class HotKey: NSObject {
+public class HotKey: NSObject {
```
Comment (Econa77) [inline Lib/Magnet/HotKey.swift]: Is there a use case to change `public` to `open` for mocking?

Comment (jasonm23) [reply Lib/Magnet/HotKey.swift]: Good point I'll make the change

**Comment by jasonm23** [comment]
Checks cannot pass due to dependabot security PRs outstanding

**Comment by jasonm23** [comment]
```
bundler: failed to load command: danger (/Users/runner/work/Sauce/Sauce/vendor/bundle/ruby/3.2.0/bin/danger)
```

**Comment by jasonm23** [comment]
/Users/runner/work/Magnet/Magnet/vendor/bundle/ruby/3.2.0/gems/octokit-6.1.1/lib/octokit/response/raise_error.rb:14:in `on_complete': POST https://api.github.com/repos/Clipy/Magnet/pulls/87/comments: 403 - Resource not accessible by integration // See: https://docs.github.com/rest/pulls/comments#create-a-review-comment-for-a-pull-request (Octokit::Forbidden)

Token permissions issue?

**Comment by jasonm23** [comment]
Looks to be cache issue

**Comment by jasonm23** [comment]
@Econa77 same token issue here

**Comment by Econa77** [comment]
@jasonm23 Added the same changes as `Sauce`. Could you please rebase it?

**Comment by jasonm23** [comment]
👍 done

**Review by Econa77 — APPROVED** [review]
👍

**Pull request merged**